src/test/java/api/requests/scenarios/SingleOpenDeliveryRequestTests.java

```diff
@@ -97,15 +119,70 @@ public void itemCanBeCheckedOutToAnotherPatron()
 
     loansFixture.checkInByBarcode(smallAngryPlanet);
 
-    loansFixture.checkOutByBarcode(smallAngryPlanet, rebecca);
+    validateRequestStatusAndPosition(requestByJessica, OPEN_AWAITING_DELIVERY, 1);
+    validateItemStatus(smallAngryPlanet, AWAITING_DELIVERY);
 
-    Response request = requestsClient.getById(requestByJessica.getId());
+    loansFixture.checkInByBarcode(smallAngryPlanet);
+
+    validateRequestStatusAndPosition(requestByJessica, OPEN_AWAITING_DELIVERY, 1);
+    validateItemStatus(smallAngryPlanet, AWAITING_DELIVERY);
+  }
+
+  @Test
+  public void itemWithCanceledRequestBecomesAvailable() throws InterruptedException,
+    MalformedURLException,
+    TimeoutException,
+    ExecutionException {
+    IndividualResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    IndividualResource james = usersFixture.james();
+    IndividualResource jessica = usersFixture.jessica();
+
+    loansFixture.checkOutByBarcode(smallAngryPlanet, james);
+
+    IndividualResource requestByJessica = requestsFixture.placeDeliveryRequest(
+      smallAngryPlanet, jessica, new DateTime(2017, 7, 22, 10, 22, 54, DateTimeZone.UTC));
+
+    requestsFixture.cancelRequest(requestByJessica);
+
+    loansFixture.checkInByBarcode(smallAngryPlanet);
```

Comment:
Is the check in required for the item to become available, or does it happen when the request is cancelled?